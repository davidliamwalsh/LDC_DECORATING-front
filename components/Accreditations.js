import { Component } from 'react'

class Accreditations extends Component {
  render () {
    return <>
      <div className="c-accreditations">
        <ul className="c-accreditations__items">
          <li className="c-accreditations__items--item">
            <img className="c-accreditations__items--item-img" src="/static/1.jpg" alt=""/>
          </li>
          <li className="c-accreditations__items--item">
            <img className="c-accreditations__items--item-img" src="/static/1.jpg" alt=""/>
          </li>
          <li className="c-accreditations__items--item">
            <img className="c-accreditations__items--item-img" src="/static/1.jpg" alt=""/>
          </li>
          <li className="c-accreditations__items--item">
            <img className="c-accreditations__items--item-img" src="/static/1.jpg" alt=""/>
          </li>
          <li className="c-accreditations__items--item">
            <img className="c-accreditations__items--item-img" src="/static/1.jpg" alt=""/>
          </li>
        </ul>
      </div>
    </>
  }
}

export default Accreditations