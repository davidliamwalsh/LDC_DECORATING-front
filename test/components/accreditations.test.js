import Accreditations from '../../components/Accreditations'

describe('<Accreditations />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <Accreditations />
    )

    expect(component).toMatchSnapshot()
  })
})