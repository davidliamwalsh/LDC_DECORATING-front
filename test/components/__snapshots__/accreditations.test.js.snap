// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Accreditations /> it renders correctly 1`] = `ShallowWrapper {}`;
